### ISSUE TITLE
email_field id doesn't propagate to label

### ISSUE THREAD

**Issue by typeoneerror** [bootstrap-ruby/rails-bootstrap-forms#213]
If you use a custom id with email_field the label's "for" is still the default:

```
= bootstrap_form_for :purchase do |f|
  = f.email_field :email, id: "my_custom_id"
```

result:

```
<div class="form-group">
  <label class="control-label" for="purchase_email">Email</label>
  <input id="my_custom_id" class="form-control" type="email" name="purchase[email]">
</div>
```

expected:

```
<div class="form-group">
  <label class="control-label" for="my_custom_id">Email</label>
  <input id="my_custom_id" class="form-control" type="email" name="purchase[email]">
</div>
```

**Comment by datWav** [comment]
I wrote fix for this bug #221

**Comment by mattbrictson** [comment]
Cleaning up old unanswered issues. If this question is still relevant, please reopen.
